src/clj/game/core/access.clj

```diff
@@ -1066,84 +1135,56 @@
             :value bonus})]
      floating-effect)))
 
-(defn access-bonus-count
-  [state side s]
-  (sum-effects state side nil :access-bonus [s]))
-
 (defn access-count
   [state side kw]
-  (let [run (:run @state)
-        s (case kw
+  (let [s (case kw
```

Comment:
I think this is unnecessary because R&D Interface et al now say `:rd` instead of `:rd-access`.